<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabBarAutoHide" stdset="0">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_hello">
       <attribute name="title">
        <string>Hello</string>
       </attribute>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>681</width>
          <height>261</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>Hello, 
welcome to the Error-Detector, 
in the next window you will need to choose 
the configuration for that session.
to continue click 'NEXT'. </string>
        </property>
       </widget>
       <widget class="QPushButton" name="nextButton1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>340</y>
          <width>85</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>NEXT</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_option_menu">
       <attribute name="title">
        <string>Coniguration</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1161</width>
          <height>481</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>5</horstretch>
          <verstretch>5</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>500</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>60</number>
         </property>
         <item row="3" column="1">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Choose your configurations:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="pushButton_stop">
           <property name="text">
            <string>STOP</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="playButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>play</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter Your name please:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="3">
          <widget class="QLineEdit" name="user_name_field">
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>options</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <widget class="QCheckBox" name="speed_checkBox">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>30</y>
              <width>151</width>
              <height>17</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>speed </string>
            </property>
           </widget>
           <widget class="QCheckBox" name="direction_checkBox">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>50</y>
              <width>141</width>
              <height>17</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Direction</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="size_checkBox">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>70</y>
              <width>70</width>
              <height>17</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_calib">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>601</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPlainTextEdit" name="plainTextEdit_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>421</width>
           <height>131</height>
          </rect>
         </property>
         <property name="plainText">
          <string>KOSTA You are about to start a calibration operation -
Follow the orders on the screen.
To start hit the button below.!!!!!</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="Button_startCalib">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>360</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Start calibration</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1175</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
